pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -823,3 +823,15 @@ def test_malformed_second_line(all_parsers):
     result = parser.read_csv(StringIO(data), skip_blank_lines=False, header=1)
     expected = DataFrame({"a": ["b"]})
     tm.assert_frame_equal(result, expected)
+
+
+def test_read_table_posargs_deprecation(all_parsers):
+    # https://github.com/pandas-dev/pandas/issues/41485
+    f = StringIO("a\tb\n1\t2")
```

Comment:
can you not use a single-letter variable name? other than that, looks good

Reply:
ok I pushed it to concat 's PR

Reply:
could you push it here instead and keep the two PRs separate?